contributor-growth/README.md

```diff
@@ -0,0 +1,65 @@
+# Contributor Growth
+
+The Contributor Growth subproject is interested in assisting CNCF projects with
+sustainably growing their contributor base.
+
+## Goals
+
+* Provide best practices and resources for managing and interacting with
+  contributors to CNCF projects.
+* Provide guidance on building a contributor pipeline that brings in new
```

Comment:
Do we want to consider providing guidance on how to handle transitioning maintainers to emeritus status? It's not so much for growth but it can affect the sustainability of contributing.